lib/querystring.js

```diff
@@ -267,7 +304,14 @@ function addKeyVal(obj, key, value, keyEncoded, valEncoded, decode) {
   }
 }
 
-// Parse a key/val string.
+/**
+ * Parse a key/val string.
+ * @param {string} qs 
+ * @param {string} [sep]
+ * @param {string} [eq] 
```

Comment:
```suggestion
 * @param {string} sep
 * @param {string} eq
```

Reading the implementation it does not look like `sep` or `eq` are optional, they always are stringified.